src/gui/processing/models/qgsmodelgraphicsscene.cpp

```diff
@@ -224,6 +224,11 @@ void QgsModelGraphicsScene::createItems( QgsProcessingModelAlgorithm *model, Qgs
     const QMap<QString, QgsProcessingModelOutput> outputs = it.value().modelOutputs();
     QMap< QString, QgsModelComponentGraphicItem * > outputItems;
 
+    // offsets from algorithm item needed to correctly place output items
+    // which does not have valid position assigned (https://github.com/qgis/QGIS/issues/48132)
+    double output_offset_x = mChildAlgorithmItems[it.value().childId()]->component()->size().width();
+    double output_offset_y = 1.5 * mChildAlgorithmItems[it.value().childId()]->component()->size().height();
```

Comment:
```suggestion
    const double outputOffsetY = 1.5 * mChildAlgorithmItems[it.value().childId()]->component()->size().height();
```

Reply:
We can't mark `outputOffsetY` as `const` as it will be modified later in line 248 to allow correct placement of multiple outputs (if algorithm has more than one).